NorthstarDLL/server/auth/serverauthentication.cpp

```diff
@@ -62,6 +62,22 @@ void ServerAuthenticationManager::StartPlayerAuthServer()
 						return;
 					}
 
+					uint64_t uid;
+					try
+					{
+						uid = std::strtoull(request.get_param_value("id").c_str(), nullptr, 10);
+					}
+					catch (std::exception const& ex)
+					{
+						response.set_content("{\"success\":false}", "application/json");
+						return;
+					}
+					if (!g_pBanSystem->IsUIDAllowed(uid))
+					{
+						response.set_content("{\"success\":false,\"reject\":\"Banned from server.\"}", "application/json");
```

Comment:
```suggestion
						response.set_content("{\"success\":false,\"reject\":\"Banned from this server.\"}", "application/json");
```

Would slightly change the phrasing to ensure that we don't have players thinking they received a global Northstar ban (which doesn't exists but they may not know that). Proper localisation is separate PR anyway as discussed.

Reply:
Changed in both places.